alsongDalsong/ASRepository/ASRepository/Repositories/PlayersRepository.swift

```diff
@@ -10,7 +10,12 @@ final class PlayersRepository: PlayersRepositoryProtocol {
     init(mainRepository: MainRepositoryProtocol) {
         self.mainRepository = mainRepository
     }
-    
+
+    func getMyID() -> String? {
+        guard let myId = ASFirebaseAuth.myID else { return nil }
+        return myId
+    }
```

Comment:
어차피 `String?`로 return 한다면 아래와 같이 더 깔끔하게 작성하는 것도 좋아보입니다!

```suggestion
    func getMyID() -> String? {
        ASFirebaseAuth.myID
    }
```

Reply:
반영했습니다!